Sources/LiveViewNative/Utils/ContentMode.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ContentMode.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/28/23.
+//
+
+import SwiftUI
+
+/// The size classes that apply to controls.
+///
+/// Possible values:
+/// * `fill`
+/// * `fit`
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+extension ContentMode: Decodable {
+    public init(from decoder: Decoder) throws {
+        let value = try decoder.singleValueContainer().decode(String.self)
+        switch value {
+        case "fill":
+            self = .fill
+        case "fitty":
+            self = .fit
+        default:
+            throw DecodingError.dataCorrupted(.init(codingPath: decoder.codingPath, debugDescription: "Unknown \(String(describing: decoder.codingPath.last)): \(value)"))
+        }
+    }
+}
```

Comment:
An example of moving the decoding of a Swift value type out of the modifier and into a type extension.